hieradata/common.yaml

```diff
@@ -30,6 +30,7 @@ psick::firstrun::windows_reboot: false
 psick::pre::linux_classes:
```

Comment:
Sorry for late feedback, I'm abroad travelling and haven't checked github for some days.